puppet/modules/quickstack/manifests/dellsc/volume.pp

```diff
@@ -1,4 +1,4 @@
-define quickstack::dell_sc_iscsi::volume (
+define quickstack::dellsc::volume (
```

Comment:
we expect that in longer term we will have both eql and sc under dell. Thus, I want correct naming conversion to handle it. Leave particular to you.